src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -235,7 +245,7 @@ export const TradeInput = (props: CardProps) => {
                   )
                 }
               >
-                {activeQuote && (
+                {sortedQuotes.length && (
```

Comment:
Heyy this is the guy. Needs to be made a boolean, this will evaluate to `0` on the left side, which is a perfectly valid react node https://legacy.reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored

Reply:
345239173f02c098e86f9cb236c14ec0936fd520